codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/RuntimeTypes.kt

```diff
@@ -9,27 +9,51 @@ import software.amazon.smithy.codegen.core.Symbol
 import software.amazon.smithy.model.node.ObjectNode
 import software.amazon.smithy.model.traits.TimestampFormatTrait
 import software.amazon.smithy.rust.codegen.rustlang.CargoDependency
+import software.amazon.smithy.rust.codegen.rustlang.CratesIo
+import software.amazon.smithy.rust.codegen.rustlang.DependencyLocation
 import software.amazon.smithy.rust.codegen.rustlang.InlineDependency
+import software.amazon.smithy.rust.codegen.rustlang.Local
 import software.amazon.smithy.rust.codegen.rustlang.RustDependency
 import software.amazon.smithy.rust.codegen.rustlang.RustType
 import software.amazon.smithy.rust.codegen.rustlang.RustWriter
 import software.amazon.smithy.rust.codegen.rustlang.asType
 import java.util.Optional
 
-data class RuntimeConfig(val cratePrefix: String = "smithy", val relativePath: String = "../") {
+sealed class RuntimeCrateLocation {
+    data class Relative(val relativePath: String) : RuntimeCrateLocation()
```

Comment:
Should this maybe be called `Path` instead? It feels like that's the operative description, as opposed to whether the path is relative or absolute.

Reply:
yeah agreed, especially since the tests use an absolute path 😂